### PR TITLE
Fix sccache statistics

### PR DESCRIPTION
**Pull request by General-Beck**
small fixes

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -63,21 +63,21 @@ cargo-check 0 3:
...
-    - sccache --stop-server
+    - sccache -s
 
 cargo-check 2 3:
   stage:                           test
   <<:                              *docker-cache-status
   script:
     - time cargo check --target $CARGO_TARGET --locked --manifest-path util/io/Cargo.toml --features "mio" --verbose --color=always
-    - sccache --stop-server
+    - sccache -s
```
Comment (dvdplm) [inline .gitlab-ci.yml]: What's wrong with `--stop-server`?

Comment (General-Beck) [reply .gitlab-ci.yml]: i's  not work in darwin. and this is more correct

Comment (General-Beck) [reply .gitlab-ci.yml]: -s show stats only. --stop-server - stop server and show stats

Comment (dvdplm) [reply .gitlab-ci.yml]:
```suggestion
    - sccache --show-stats
```

More readable imo!

**Pull request merged**